core/pkg/eval/fractional_evaluation_test.go

```diff
@@ -63,46 +63,99 @@ func TestFractionalEvaluation(t *testing.T) {
 		expectedReason  string
 		expectedError   error
 	}{
-		"test@faas.com": {
+		"rachel@faas.com": {
```

Comment:
I changed the emails so that we get all the colours. Probably a waste of time as it will change if #846 gets merged 🙃

Reply:
> I changed the emails so that we get all the colours. Probably a waste of time as it will change if #846 gets merged 🙃

I like the idea though, lol

Reply:
We still get all the colours :metal: